cmd/util.go

```diff
@@ -124,3 +129,64 @@ func ListSubnets(region string, vpcId string, az string) []string {
 	return subnets
 
 }
+
+func PutFilesToS3(region string, bucketName string, path string, filenames []string) {
+	svc := s3.New(session.New(), &aws.Config{Region: aws.String(region)})
+
+	for _, filename := range filenames {
+		file, err := os.Open(fmt.Sprintf("%s%s", path, filename))
+		if err != nil {
+			log.Panic(err.Error())
+		}
+		defer file.Close()
+
+		resp, err := svc.PutObject(&s3.PutObjectInput{
+			Bucket: aws.String(bucketName),
+			Key:    aws.String(filename),
+			Body:   file,
+		})
+
+		if err != nil {
+			log.Panic(err.Error())
+		}
+
+		log.Println(awsutil.StringValue(resp))
+	}
+}
+
+func GetFilesFromS3(region string, bucketName string, path string, filenames []string) {
+	downloader := s3manager.NewDownloader(session.New(&aws.Config{Region: aws.String(region)}))
+
+	for _, filename := range filenames {
+		file, err := os.Create(fmt.Sprintf("%s%s", path, filename))
+		if err != nil {
+			log.Panic("Failed to create file", err)
+		}
+		defer file.Close()
+
+		numBytes, err := downloader.Download(file, &s3.GetObjectInput{
+			Bucket: aws.String(bucketName),
+			Key:    aws.String(filename),
+		})
+
+		if err != nil {
+			log.Panic(err.Error())
+		}
+
+		log.Println("Downloaded file: ", file.Name(), " (", numBytes, "bytes)")
+	}
+}
+
+func completeCfgDir(filename string) string {
```

Comment:
As far as I can see, this function returns a more complete path of a file, not a directory.
But `completeCfgDir` sounds like a function returning the path to a directory if you take `complete` as an adjective, not a verb as you've probably intended.

I propose `prefixConfigDir` for its name so it might be clearer that it just prefixes a filename with the path of the configuration directory.